web/src/lib/components/album-page/albums-list.svelte

```diff
@@ -0,0 +1,331 @@
+<script lang="ts" context="module">
+  import { AlbumViewMode, albumViewSettings, locale } from '$lib/stores/preferences.store';
+  export interface Sort {
+    title: string;
+    sortDesc: boolean;
+    widthClass: string;
+    sortFn: (reverse: boolean, albums: AlbumResponseDto[]) => AlbumResponseDto[];
+  }
+
+  export let sortByOptions: Record<string, Sort> = {
```

Comment:
We always iterate over this object to match on `title`, and never use the key for lookups. We should change this so the key can be used, or just convert it to an array.